src/map/mob.cpp

```diff
@@ -5563,7 +5563,7 @@ static void mob_load(void)
 
 		sv_readdb(dbsubpath1, "mob_chat_db.txt", '#', 3, 3, -1, &mob_parse_row_chatdb, silent);
 
-		if( db_use_sqldbs && i == 0 )
+		if( db_use_sqldbs )
```

Comment:
Need this `if( db_use_sqldbs && i == 0 )`

`[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1150[MOONLIGHT]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1511[AMON_RA]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1637[MAGALETA]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1639[KATRINN]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1643[G_MAGALETA]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1646[B_SEYREN]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1647[B_EREMES]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1649[B_MAGALETA]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1650[B_SHECIL]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1651[B_KATRINN]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1719[DETALE]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1734[KIEL_]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1751[RANDGRIS]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1779[KTULLANUX]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1870[NECROMANCER]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1871[FALLINGBISHOP]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1874[BEELZEBUB_]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1916[MOROCC]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1917[MOROCC_]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1929[BAPHOMET_I]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1947[PIAMETTE_]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1956[NAGHT_SIEGER]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 1990[HARDROCK_MOMMOTH]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2009[WOOMAWANG_]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2022[S_NYDHOG]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2103[E_KTULLANUX]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2107[E_FALLINGBISHOP]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2109[E_RANDGRIS]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2112[E_B_SEYREN]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2113[E_B_EREMES]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2131[LOST_DRAGON]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2156[LEAK]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2189[COELACANTH_H_M]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2190[COELACANTH_H_A]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2194[MD_GIANT_OCTOPUS]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2202[KRAKEN]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2223[CELIA]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2228[G_RANDEL]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2230[G_CELIA]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2232[G_GERTIE]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2235[B_RANDEL]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2236[B_FLAMEL]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2237[B_CELIA]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2238[B_CHEN]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2239[B_GERTIE]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2240[B_ALPHOCCIO]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2241[B_TRENTINI]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2362[N_AMON_RA]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2419[L_KATRINN]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2432[G_L_KATRINN]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2736[C3_NECROMANCER]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2875[C3_CELIA]
[Error]: mob_parse_row_mobskilldb: Too many skills for monster 2999[EP14_MORS_BOSSA]`